code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -517,10 +533,13 @@
 
 
 	//Shooting Code:
-	A.preparePixelProjectile(target, src)
+	A.preparePixelProjectile(target, T)

```

Comment:
does this not by itself fix it?
(I think dense objects can EXIT other dense objects...)

Reply:
The issue is not with exiting the turf the turret is on but hitting the surrounding turfs on cardinal directions.